site/app/libraries/Output.php

```diff
@@ -34,6 +37,12 @@ class Output {
     public function __construct(Core $core) {
         $this->core = $core;
         $this->start_time = microtime(true);
+
+        $this->twig_loader = new \Twig_Loader_Filesystem(FileUtils::joinPaths(dirname(__DIR__), 'templates'));
```

Comment:
So we could have the options work like the following:
```
$options = [];
if (!$this->core->getConfig->isDebug()) {
    $options['cache'] = FileUtils::joinPaths(dirname(__DIR__), 'cache', 'templates');
}
```

However, @bmcutler, this would require the cache/templates folder being read/write and I'm not sure the permissions on the cached templates would match everything? We could implement/add something like [twig-api](https://github.com/farazdagi/twig-cli) and try and compile all of the templates in the installation process and then we can set the permissions as we like.